### ISSUE TITLE
Error on Cache disabled

### ISSUE THREAD

**Issue by SamMnrve** [jeremyharris/cacher#17]
if you disable the cache, the variable source is a null when it does read in the datasource.

if i move the line:
$this->source = ConnectionManager::getDataSource($this->config['original']);
before the test:
if (Configure::read('Cache.disable') === true) {

it's OK

**Comment by jeremyharris** [comment]
@SamMnrve yeah, historically Cacher hasn't been happy when the cache has been disabled app-wide. I'll go through and write some tests and make Cacher behave.
